README.md

```diff
@@ -344,6 +349,10 @@ And all docs are available here: [release-please docs](https://github.com/google
 
 Use [Conventional Commits](https://www.conventionalcommits.org/) to ensure that the changelogs are generated correctly.
 
+### Releasable units
+
+Release please goes through commits and tries to find "releasable units" using commit messages as guidance - it will then add these units to their respective release PR's and figures out the version number from the types: `fix` for patch, `feat` for minor, `feat!` for major. All the other types will not be included in the changelog. So, you can use for example `chore` or `refactor` to do work that does not need to be included in the changelog and won't bump the version.
```

Comment:
How would you feel about this change?
"All the other types will not be included in the changelog." → "None of the other types will be included in the changelog unless they are [language specific releaseable units](https://github.com/googleapis/release-please#release-please-bot-does-not-create-a-release-pr-why)". Or if you don't want to go into the language specific releaseable units then just "All the other types will not be included in the changelog." → "None of the other types will be included in the changelog."?

Reply:
"None of the other types will be included in the changelog" might be better, so not to confuse people with the language spesific -things.